### PR TITLE
Added space between Chinese and English words.

### LINKED ISSUE #15843: [OpenForce 2022] Optimize official documents
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Aim

Check the source file below, keep a space between Chinese and English words

Source file:
https://github.com/apache/shardingsphere/blob/master/docs/community/content/contribute/code-conduct.cn.md

Official document link:
https://shardingsphere.apache.org/community/cn/contribute/code-conduct/

### Example

`您可以报告bug，提交一个新的功能增强建议或者直接对以上内容提交改进补丁。`

After optimization：

`您可以报告 bug，提交一个新的功能增强建议或者直接对以上内容提交改进补丁。`

### PR DESCRIPTION
**Pull request by jhanvi831**
Fixes #15843 

Added required spaces between Chinese and English words.

### CODE REVIEW

File: docs/community/content/contribute/code-conduct.cn.md [hunk 1]
```diff
@@ -19,74 +19,74 @@ chapter = true
...
-   - 公用规则定义放在`Keyword.g4`，每个数据库可以有自己特有的规则定义。例如：`MySQLKeyword.g4`。
+   - 规则名称使用大写字母。如果名称由多个单词组成，用`下划线`间隔。`DataType` 和 `Symbol` 的规则命名以 `下划线` 结尾。与 ANTLR 内置变量或关键字重名的规则在结尾加`下划线`以示区分。
+   - 不对外暴露的规则使用 `fragment`，`fragment` 定义的规则需在其服务的规则之后声明。
+   - 公用规则定义放在 `Keyword.g4`，每个数据库可以有自己特有的规则定义。例如：`MySQLKeyword.g4`。
  - 语法解析规范
    - 每个规则结束后空一行，空行无需缩进。
    - 规则名称前面不空格，`冒号`后空一格再开始写规则，`分号`在单独一行并保持和上一行相同缩进。
-   - 如果一个规则的分支超过`5`个，则每个分支一行。
+   - 如果一个规则的分支超过 `5` 个，则每个分支一行。
    - 规则命名采用java变量的驼峰形式。
```
Comment (menghaoranss) [inline docs/community/content/contribute/code-conduct.cn.md:91]: The last 2 lines may be revised together.

Comment (jhanvi831) [reply docs/community/content/contribute/code-conduct.cn.md]: Thank you for letting me know, I corrected it as well.

Comment (jhanvi831) [reply docs/community/content/contribute/code-conduct.cn.md]: Can you confirm if its visible to you as well ?

Comment (menghaoranss) [reply docs/community/content/contribute/code-conduct.cn.md]: @jhanvi831 perfect, thanks for your contribution.

**Review by menghaoranss — APPROVED** [review]

**Pull request merged**